src/Compilers/Core/Portable/Operations/BasicBlock.cs

```diff
@@ -59,7 +38,7 @@ public BasicBlock(BasicBlockKind kind)
         /// <summary>
         /// PROTOTYPE(dataflow): Tuple is temporary return type, we probably should use special structure instead.
         /// </summary>
-        public (IOperation Value, ConditionalBranchKind Kind, BasicBlock Destination) Conditional { get; internal set; }
+        public (IOperation Condition, bool JumpIfTrue, BasicBlock Destination) Conditional { get; internal set; }
```

Comment:
@jaredpar Would it be difficult to get some protection in place to prevent accidentally shipping a publicly-exposed tuple return.